lnd_test.go

```diff
@@ -39,6 +39,7 @@ import (
 
 var (
 	harnessNetParams = &chaincfg.SimNetParams
+	lndHarness       *lntest.NetworkHarness
```

Comment:
Do you really need to define this as a var? see comment below

Reply:
I think I need to define this as a var to use it in Fatalf function.

Reply:
You can make it a part of the `harnessTest` struct.